### PR TITLE
SND-647 Improve naming of roles

### PR DESCRIPTION
**Pull request by oleksiidn**

### CODE REVIEW

**Review by jhyoty — APPROVED** [review]

File: src/main/java/global/govstack/usct/configuration/SecurityConfig.java [hunk 1]
```diff
@@ -253,10 +253,10 @@ private static Collection<? extends GrantedAuthority> mapAuthorities(
         log.info("Logging in user {}", subject);
         return switch (subject) {
           case "299950323465436931629862208523254959" -> Set.of(
-              new SimpleGrantedAuthority("ROLE_" + ENROLLMENT_OFFICER));
```
Comment (valentin-filyov) [inline src/main/java/global/govstack/usct/configuration/SecurityConfig.java:256]: I think those changes are not needed

File: docs/main.md [hunk 2]
```diff
@@ -55,11 +55,11 @@ To trigger the OIDC authentication flow, a client should issue a GET request to
 Local authentication uses username/password 
 Mosip uses Foundational ID (VID)
 
-| VID / username                       | Role               | subject                              | Description                                         |
-|--------------------------------------|--------------------|--------------------------------------|-----------------------------------------------------|
-| 5649650687 / registry-administration | ENROLLMENT_OFFICER | 299950323465436931629862208523254959 | Officer responsible for enrollment                  |
```
Comment (valentin-filyov) [inline docs/main.md:60]: 564965068 is for enrollment officer
4893724702 is for payment officer
2371487382 is for registry officer

**Review by valentin-filyov — COMMENTED** [review]
I think roles in the second column of the first table were fine so did the roles in securityconfig

File: src/main/java/global/govstack/usct/configuration/SecurityConfig.java (hunk 1 above)
Comment (oleksiidn) [reply src/main/java/global/govstack/usct/configuration/SecurityConfig.java:256]: I tried to follow the logic that we have in the documentation.
1. Create candidate
2. Enroll 
3. Payment 

`Copy/paste` from the documentation.

File: docs/main.md (hunk 2 above)
Comment (oleksiidn) [reply docs/main.md:60]: Implemented it 
https://github.com/GovStackWorkingGroup/sandbox-usecase-usct-backend/pull/44

**Pull request merged**